examples/problems/1200_conditional_entropy_variance.ipynb

```diff
@@ -0,0 +1,387 @@
+{
```

Comment:
<p>that's great. now just add a little bit of explanation to the technical side, e.g. what does <code>forward</code> and <code>batch_size</code> mean  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>